docs/docusaurus/docs/guides/expectations/create_expectations_overview.md

```diff
@@ -14,57 +14,57 @@ import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
 
-As with creating Expectations interactively, you will start with your Data Context.  However, in this case you will be working in a Python environment, so you will need to load or create your Data Context as an instantiated object.  Next, you will create a Batch Request to specify the data you would like to <TechnicalTag tag="profiling" text="Profile" /> with your Data Assistant.  Once you have a <TechnicalTag tag="batch_request" text="Batch Request" /> configured you will use it as the input for the run method of your Data Assistant, which can be accessed from your Data Context object.  Once the Data Assistant has run, you will be able to review the results as well as save the generated Expectations to an empty Expectation Suite.
+### Create Expectations with custom methods
 
-The Data Assistant we recommend using for new data is the Onboarding Data Assistant.  We provide an in-depth guide to this in our documentation on [how to create an Expectation Suite with the Onboarding Data Assistant](./data_assistants/how_to_create_an_expectation_suite_with_the_onboarding_data_assistant.md).
+Advanced users can use custom methods to generate Expectations that are based on source data system metadata. If you want to use custom methods to create Expectations, [contact a member of the support team on Slack](https://greatexpectations.io/slack).
 
-### Testing your Expectation Suite
+### Test your Expectation Suite
 
-Once you have created your Expectation Suite and saved it, you may wish to test it.  The simplest way to do this is to Validate some data against it.  You can do this using a `SimpleCheckpoint` as demonstrated in the [optional step on running validation, saving your suite, and building Data Docs](./how_to_create_and_edit_expectations_with_a_profiler.md#6-optional-running-validation-saving-your-suite-and-building-data-docs) of our [how to create and edit Expectations with a Profiler](./how_to_create_and_edit_expectations_with_a_profiler.md) documentation.  Or you can just move on to [Step 4: Validate Data.](../validation/validate_data_overview.md)
+After you've created and saved your Expectation Suite, GX recommends that you test it by validating it. You can validate an Expectation with `SimpleCheckpoint` as demonstrated [here](./how_to_create_and_edit_expectations_with_a_profiler.md#6-optional-running-validation-saving-your-suite-and-building-data-docs). An overview of the validation process is provided [here](../validation/validate_data_overview.md).
 
-### Editing a saved Expectation Suite
+### Edit a saved Expectation Suite
```

Comment:
@Rachel-Reverie rather than repeat existing information (and add a requirement to edit information in two locations in future), I removed the existing content and added a link to _How to Edit an Expectation Suite_.